propka/bonds.py

```diff
@@ -1,336 +1,334 @@
-
-from __future__ import division
-from __future__ import print_function
-
-import pickle,sys,os,math,propka.calculations
+"""PROPKA representation of bonds."""
+# TODO - is pickle still used?
+import pickle
+# TODO - eliminate use of sys
+import sys
+# TODO - eliminate use of os
+import os
+import math
 import json
-
 import pkg_resources
+import propka.calculations
+# TODO - replace the info/warning imports with logging functionality
```

Comment:
yes, please – but can be a separate issue/PR

Reply:
Yes, I created #39 as an issue to track this.